config/options.go

```diff
@@ -155,10 +155,6 @@ type Options struct {
 	// https://openid.net/specs/openid-connect-basic-1_0.html#RequestParameters
 	RequestParams map[string]string `mapstructure:"idp_request_params" yaml:"idp_request_params,omitempty"`
 
-	// Administrators contains a set of emails with users who have super user
-	// (sudo) access including the ability to impersonate other users' access
-	Administrators []string `mapstructure:"administrators" yaml:"administrators,omitempty"`
```

Comment:
We should add a breaking item to the UPGRADE guide / changelog and update the docs?